test.js

```diff
@@ -19,7 +19,7 @@ test('main', async t => {
 });
 
 test('passes arguments through', async t => {
-	const throttled = m(async x => x, 1, 100);
+	const throttled = m(async x => x, 1, 100); // eslint-disable-line require-await
```

Comment:
This seems like a false-positive in XO?

Reply:
Yes, but it's impossible to check whether the return value is expected to be a promise even without an `await`.